lib/device/library/windows.rb

```diff
@@ -3,32 +3,44 @@
 # Copyright 2013 whiteleaf. All rights reserved.
 #
 
+# rubocop:disable Style/VariableName
+
 require "win32ole"
 require_relative "../../extensions/windows"
 
-module Device::Library
-  module Windows
-    def get_device_root_dir(volume_name)
-      @@FileSystemObject ||= WIN32OLE.new("Scripting.FileSystemObject")
-      get_drives.each do |drive_letter|
-        drive_info = @@FileSystemObject.GetDrive(drive_letter)
-        vol = drive_info.VolumeName rescue ""
-        if vol.downcase == volume_name.downcase
-          return File.expand_path(drive_letter)
+class Device
+  module Library
+    module Windows
+      def get_device_root_dir(volume_name)
+        @@FileSystemObject ||= WIN32OLE.new("Scripting.FileSystemObject")
+        drives.each do |drive_letter|
+          drive_info = @@FileSystemObject.GetDrive(drive_letter)
+          vol = drive_info.VolumeName rescue ""
+          if vol.casecmp(volume_name) == 0
+            return File.expand_path(drive_letter)
+          end
         end
+        nil
+      end
+
+      def drives
+        result = []
+        bits = WinAPI.GetLogicalDrives
+        26.times do |i|
```

Comment:
__[reek]__
* [UncommunicativeVariableName] has the variable name 'i' [:ref](https://github.com/troessner/reek/blob/master/docs/Uncommunicative-Variable-Name.md)